app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,11 @@
   </head>
 
   <body>
+  <% flash.each do |type, msg| %>
+    <div>
+      <%= msg %>
+    </div>
+  <% end %>
```

Comment:
Great use of building this in `application.html.erb` file so others can inherit from it.